packages/sdk/contractkit/src/celo-tokens.ts

```diff
@@ -0,0 +1,201 @@
+import { BigNumber } from 'bignumber.js'
+import { CeloContract, CeloTokenContract, ExchangeContract, StableTokenContract } from './base'
+import { ContractKit } from './kit'
+import { GoldTokenWrapper } from './wrappers/GoldTokenWrapper'
+import { StableTokenWrapper } from './wrappers/StableTokenWrapper'
+
+export enum StableToken {
+  cUSD = 'cUSD',
+  cEUR = 'cEUR',
+}
+
+export enum Token {
+  CELO = 'CELO',
+}
+
+export type CeloToken = StableToken | Token
```

Comment:
(sorry missed to add the concern, haha)
Basically is the name collision between this type and the one in base (even that we know that the other will be deprecated). This could eventually bring some confusion.

Reply:
Just changed to `CeloTokenType`